gammapy/cube/core.py

```diff
@@ -324,6 +320,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap along slicepos`.
```

Comment:
The closing backtick is in the wrong place here ... it should be:
```
Slice out a 2-dim `~gammapy.image.SkyMap` at a given energy slice position.
```

One important question is whether this method should create a "view" or a "copy".

A view is more efficient, but this can also be very surprising to users if they create an image this way, modify it, and then the underlying cube changes.
I think the default behaviour should be the "safe version" of `copy=True` to create a decoupled image, and then there should be an option `copy=False` that one can use for the few cases where the copy is too costly and for experts.
@adonath - Do you agree?

Also -- the current behaviour of this method is not clear to me. I'm pretty sure name and wcs are by reference (i.e. coupled, not a copy). For data, I think `self.data[slicepos]` is a view and 
This will produce a "view", i.e. create a `SkyMap` where the `name`, `data`, `unit` and `Quantity(self.data[slicepos], self.data.unit)` also doesn't make a copy ... but here I'm not sure.

@OlgaVorokh - Could you play with this (or an equivalent little test case with a Quantity in IPython) a little until you understand when a copy is made and when it isn't, and then add the `copy=False` option to this method and implement it?
I don't think the behaviour needs to be verified via tests ... this is something we can do via code review. But if you want to add a test that establishes that `copy=False` and `copy=True` work as expected, that's fine too.